src/PrestaShopBundle/Controller/ArgumentResolver/SearchParametersResolver.php

```diff
@@ -79,19 +87,29 @@ class SearchParametersResolver implements ArgumentValueResolverInterface
      * @param AdminFilterRepository $adminFilterRepository
      * @param EventDispatcherInterface $dispatcher
      * @param int $shopId The Shop id
+     * @param CommandBusInterface $queryBus
      */
     public function __construct(
         SearchParametersInterface $searchParameters,
         TokenStorageInterface $tokenStorage,
         AdminFilterRepository $adminFilterRepository,
         EventDispatcherInterface $dispatcher,
-        $shopId
+        $shopId,
+        CommandBusInterface $queryBus
```

Comment:
Technically this is a BC break 😞. @sarjon @PierreRambaud wdyt ? This service is used internally by our SF implementation, I think nobody will reuse it ?

Reply:
I've reverted the braking changes since this service is no longer used. Apparently I didn't even need to change it :slightly_smiling_face: